Stargate isn't able to load schema if Solr is used on a table

When trying to connect Stargate to the existing DSE 6.8.6 cluster there are errors

`
ERROR [MigrationStage:1] 2021-01-19 10:11:59,814 PullRequestScheduler.java:245 - Configuration exception merging remote schema
org.apache.cassandra.exceptions.ConfigurationException: Unable to find custom indexer class 'com.datastax.bdp.search.solr.Cql3SolrSecondaryIndex'
    at org.apache.cassandra.utils.FBUtilities.classForName(FBUtilities.java:493)
    at org.apache.cassandra.schema.IndexMetadata.getCustomIndexClass(IndexMetadata.java:190)
    ....
    at org.apache.cassandra.utils.concurrent.InlinedThreadLocalThread.run(InlinedThreadLocalThread.java:251)
Caused by: java.lang.ClassNotFoundException: com.datastax.bdp.search.solr.Cql3SolrSecondaryIndex not found by io.stargate.db.dse [1]
    at org.apache.felix.framework.BundleWiringImpl.findClassOrResourceByDelegation(BundleWiringImpl.java:1597)
    ... 24 common frames omitted
`

This prevents all existing tables (except of some dse system keyspaces) being worked with completely via Stargate using GraphQL, Rest API, cqlsh as Stargate cannot load the schemas if there are tables with Solr used.

--- a/persistence-dse-6.8/src/main/java/com/datastax/bdp/search/solr/Cql3SolrSecondaryIndex.java
+++ b/persistence-dse-6.8/src/main/java/com/datastax/bdp/search/solr/Cql3SolrSecondaryIndex.java
@@ -33,7 +33,7 @@ public class Cql3SolrSecondaryIndex implements Index {
 
   private final IndexMetadata indexMetadata;
 
-  public Cql3SolrSecondaryIndex(ColumnFamilyStore columnFamilyStore, IndexMetadata indexMetadata) {
+  public Cql3SolrSecondaryIndex(ColumnFamilyStore ignoredCfs, IndexMetadata indexMetadata) {
     this.indexMetadata = indexMetadata;
   }
 

